src/encoded/types/base.py

```diff
@@ -32,47 +32,56 @@ def mixin_smaht_permission_types(schema: dict) -> dict:
         'type': 'array',
         'items': {
             'linkTo': 'SubmissionCenter'
-        }
+        },
+        'serverDefault': 'user_submission_centers'
     }
     schema['properties']['consortiums'] = {
         'type': 'array',
         'items': {
-            'linkTo': 'Consortiums'
-        }
+            'linkTo': 'Consortium'
+        },
+        'serverDefault': 'user_consortiums'
     }
     return schema
 
 
-class SMAHTCollection(Collection):
+class SMAHTCollection(Collection, AbstractCollection):
     """ Allows default ACL """
     def __init__(self, *args, **kw):
-        """smth."""
         super(Collection, self).__init__(*args, **kw)
         if hasattr(self, '__acl__'):
+            PRINT(f'DEBUG_PERMISSIONS: returning {self.__acl__} for {self.type_info.name}')
             return
 
-        # If no ACLs are defined for collection, allow project members to create
+        # If no ACLs are defined for collection, allow submission centers to add/create
         if 'submission_centers' in self.type_info.factory.schema['properties']:
-            self.__acl__ = ALLOW_SUBMISSION_CENTER_MEMBER_EDIT_ACL
+            PRINT(f'DEBUG_PERMISSIONS: returning {ALLOW_SUBMISSION_CENTER_CREATE_ACL} for {self.type_info.name}')
+            self.__acl__ = ALLOW_SUBMISSION_CENTER_CREATE_ACL
+        else:
+            self.__acl__ = ONLY_ADMIN_VIEW_ACL
+            PRINT(f'DEBUG_PERMISSIONS: using admin acl for {self.type_info.name}')
 
 
 @abstract_collection(
-    name='smaht-items',
+    name='items',
```

Comment:
I don't know how I feel about renaming this but not renaming SMaHTItem. Can you elaborate on how I'm supposed to think about why that is? Maybe even document in a comment someplace, though this might not be that place.